mysql/README.md

```diff
@@ -1,59 +1,131 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog agent's MySQL check sends many database metrics to Datadog, including metrics for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And [many more](https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv). You can also invent your own metrics using custom queries.
```

Comment:
Again, we may be jumping the gun, but my vote for now:

Put a Metrics section in integrations-core READMEs which always begins "See [metadata.csv] for a full list of provided metrics" - which Docs site can easily regex out - followed by any additional comments on metrics. There *are* additional comments needed in #297 for NGINX metrics, i.e. that some differently-named metrics from NGINX Plus vs. Standard NGINX actually refer to the same metric. We'd want to include comments like this in both integrations-core README and in the Docs site (and probably in dogweb, too).

Thoughts @jyee?